sycl/source/detail/plugin.hpp

```diff
@@ -10,8 +10,8 @@
 #include <CL/sycl/backend_types.hpp>
 #include <CL/sycl/detail/common.hpp>
 #include <CL/sycl/detail/pi.hpp>
+#include <CL/sycl/detail/plugin_printers.hpp>
```

Comment:
It seems `plugin_printers.hpp` is still being moved to `include` directory in the latest version of the patch. Can we avoid doing that?

Reply:
You're right. I missed the file. I'll put the file back to where it was.